themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```diff
@@ -533,3 +533,8 @@ form#kc-select-back-form div.login-pf-social-section {
     padding-left: 0px;
     border-left: 0px;
 }
+
+.delete-account-text {
+    font-size: large;
+    font-weight: 600;
+}
```

Comment:
Remove this and use btn-danger class on button instead

Reply:
This style is not meant for the button. It is meant for the paragraph here: https://github.com/keycloak/keycloak/blob/3441357ddfb17fd84ed294a1315d64e5e5c60c21/themes/src/main/resources/theme/base/login/delete-account-confirm.ftl#L16